docker-compose.yaml

```diff
@@ -42,3 +42,16 @@ services:
       KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
       KAFKA_INTER_BROKER_LISTENER_NAME: PLAINTEXT
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+  mssql:
+    image: mcr.microsoft.com/azure-sql-edge
+    stdin_open: true
+    ports:
+      - "1433:1433"
+    environment:
+      - MSSQL_SA_PASSWORD=Asdf1234!
```

Comment:
let's use `eds` to be consistent with the others

Reply:
Let me double check on this, but I think there was some sort of password complexity requirement when I tried to run this image with a simpler password

Reply:
Yep- ran into
`ERROR: Unable to set system administrator password: Password validation failed. The password does not meet SQL Server password policy requirements because it is not complex enough. The password must be at least 8 characters long and contain characters from three of the following four sets: Uppercase letters, Lowercase letters, Base 10 digits, and Symbols..`

Reply:
maybe we set the other passwords then to be the same?

Reply:
Sounds good